src/static/bitbucket/index.ejs

```diff
@@ -11,6 +11,9 @@
           Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
       }
   </style>
+    <script>
+      window.__WIDGET_SETTINGS__ = <%= widgetSettings %>
```

Comment:
how does this work? I didn't imagine it'd be this complicated. I was imagining something like from landkid deployment for a given repo it would pass some var as a config that defaulted to some values.

This is the the way landkid handles it then it's the way it needs to be I suppose

Reply:
Landkid configuration (till now) has been only used to power server-side / build time configuration though.
This is the first time we needing to pass on values from the `config.js` file onto the frontend (without going through an API call.